pos_voucher/static/src/js/popups.js

```diff
@@ -35,7 +35,7 @@ odoo.define('pos_voucher.popups', function (require) {
                 this.options.confirm.call(
                     this,
                     $('#voucher_type_selection').val(),
-                    this.inputbuffer
+                    parseFloat(this.inputbuffer.replace(',', '.'))
```

Comment:
I think we can do better than this..

What about 
https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/js/screens.js#L1848 ?

something like this
```js
var amount = 0;
if (this.inputbuffer !== "-") {
    amount = field_utils.parse.float(this.inputbuffer);
}
```